libs/test-commons/.eslintrc.json

```diff
@@ -3,15 +3,15 @@
   "ignorePatterns": ["!**/*"],
   "overrides": [
     {
-      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "files": ["*.ts", "*.tsx", "*.ts", "*.tsx"],
       "rules": {}
     },
     {
       "files": ["*.ts", "*.tsx"],
       "rules": {}
     },
     {
-      "files": ["*.js", "*.jsx"],
+      "files": ["*.ts", "*.tsx"],
```

Comment:
There are already rules for TS files above, and you set `lintFilePatterns` to be only TS files (look in the packe.json file), so rules for JS are unnecessary